lua/nvim-treesitter/parsers.lua

```diff
@@ -601,6 +601,13 @@ list.fortran = {
   },
 }
 
+list.cmake = {
+  install_info = {
+    url = "https://github.com/uyha/tree-sitter-cmake",
+    files = { "src/parser.c"},
+  },
+}
+
```

Comment:
Could you just add you as the maintainer on this language?
As you maintain the parser, you may want to also maintain the queries here?

Reply:
I've added myself as the maintainer, and yes, I intend to maintain the queries here as well